package.json

```
@@ -69,6 +69,7 @@
     "@babel/preset-env": "7.3.4",
     "@babel/preset-react": "7.0.0",
     "@babel/runtime": "7.3.4",
+    "axios": "^0.18.0",
```

Please use exact version.
```suggestion
    "axios": "0.18.0",
```